src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementService.java

```diff
@@ -124,6 +125,7 @@ public ClaimDocument logUploadDocumentFailure(
     public byte[] downloadDocument(String authorisation, URI documentSelf, String baseFileName) {
         try {
             UserDetails userDetails = userService.getUserDetails(authorisation);
+            String userRoles = String.join(",", this.userRoles);
```

Comment:
this is unnecessary. The previous code was working fine

Reply:
we are converting to list above and then again converting to string. Please remove this redundant code.